### ISSUE TITLE
markdown-preview is not working css local files

### ISSUE THREAD

**Issue by jcbritobr** [atom/markdown-preview#266]
The preview dont show changes for css local files. This is a nice feature to have a bug that was filled in the last year.

**Comment by izuzak** [comment]
@jcbritobr Can you share a link to the bug you're referring to, and also provide the exact steps to reproduce the problem?

**Comment by jcbritobr** [comment]
Ok

Em sáb, 27 de jun de 2015 11:18, Ivan Žužak <notifications@github.com>
escreveu:

**Comment by jcbritobr** [comment]
Hi ivan, here is the link of the bug. It happens now in 1.0 linux ubuntu
15.04
https://github.com/webBoxio/atom-html-preview/issues/1

*Júlio César de Brito Gardona*
Especialista em Sistemas | Systems Specialist

[image: Perfil no LinkedIn]
<https://www.linkedin.com/profile/view?id=134240849&trk=nav_responsive_tab_profile>
[image:
Perfil no Twitter] <https://twitter.com/jcbritogardona> [image: Perfil no
Google Plus]
<https://plus.google.com/u/0/+J%C3%BAlioC%C3%A9sarBritoBr/about> [image:
Effective Craft - Engenharia, processos e desenvolvimento de software]
<http://effectivecraft.blogspot.com.br/>

2015-06-27 12:35 GMT-03:00 Júlio César Brito Gardona <jcbritobr@gmail.com>:

**Comment by mnquintana** [comment]
I'm not sure I follow - atom-html-preview is a different package from this one (which is markdown-preview).

**Comment by jcbritobr** [comment]
Yes, but is the same problem. Maybe create a new track for this issue.

*Júlio César de Brito Gardona*
Especialista em Sistemas | Systems Specialist

[image: Perfil no LinkedIn]
<https://www.linkedin.com/profile/view?id=134240849&trk=nav_responsive_tab_profile>
[image:
Perfil no Twitter] <https://twitter.com/jcbritogardona> [image: Perfil no
Google Plus]
<https://plus.google.com/u/0/+J%C3%BAlioC%C3%A9sarBritoBr/about> [image:
Effective Craft - Engenharia, processos e desenvolvimento de software]
<http://effectivecraft.blogspot.com.br/>

2015-06-27 23:08 GMT-03:00 Machisté N. Quintana <notifications@github.com>:
